src/index.html

```diff
@@ -17,16 +17,37 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search-bar"
+      data-qa="big"
+    >
```

Comment:
The 'div' element for the search bar should be inside a 'form' tag as per the checklist requirement: 'Inputs are written inside of 'form' tag with correctly passed attributes'.